backend/dataall/modules/shares_base/services/sharing_service.py

```diff
@@ -216,14 +216,14 @@ def revoke_share(cls, engine: Engine, share_uri: str) -> bool:
                                 share_uri,
                                 old_status=ShareItemStatus.Revoke_Approved.value,
                                 new_status=ShareItemStatus.Revoke_Failed.value,
-                                share_item_type=processor.type.value,
+                                share_item_type=processor.type,
                             )
                             ShareStatusRepository.update_share_item_status_batch(
                                 session,
                                 share_uri,
                                 old_status=ShareItemStatus.Revoke_In_Progress.value,
                                 new_status=ShareItemStatus.Revoke_Failed.value,
-                                share_item_type=processor.type.value,
+                                share_item_type=processor.type,
```

Comment:
PR https://github.com/data-dot-all/dataall/pull/1452/files (already merged) solves this issue